src/constants.ts

```diff
@@ -20,6 +20,7 @@ export const QUOTE_ORDER_EXPIRATION_BUFFER_MS = ONE_SECOND_MS * 60; // Ignore or
 const GAS_LIMIT_BUFFER_PERCENTAGE = 0.1; // Add 10% to the estimated gas limit
 export const GAS_LIMIT_BUFFER_MULTIPLIER = GAS_LIMIT_BUFFER_PERCENTAGE + 1;
 export const DEFAULT_QUOTE_SLIPPAGE_PERCENTAGE = 0.01; // 1% Slippage
+export const DEFAULT_PRICE_IMPACT_PROTECTION_PERCENTAGE = 0.2; // 20%
```

Comment:
[no action required] 
what's your thoughts on the unit here?
I have felt these these `_PERCENTAGE` constants are misleading.

Reply:
yeah its a little weird to say "_PERCENTAGE" and then expect .2 for 20%. I didn't think too much about it when writing it I just followed the convention. I'm up to change it if we don't like it but i'm not sure if `DEFAULT_PRICE_IMPACT_PRETOECTION` is more clear....

Reply:
Looks like we are stuck with this convention because of slippage percentage... 
i guess it's better to be consistently "wrong" than having two different styles. 😆 
Let's leave it as it is.